Model/AdyenThreeDS2Process.php

```diff
@@ -92,21 +92,23 @@ public function initiate($payload)
         } else {
             // Create order by order id
             $order = $this->orderFactory->create()->load($payload['orderId']);
+            // don't send orderId to adyen. Improve that orderId and state.data are separated in payload
+            unset($payload['orderId']);
         }
 
         $payment = $order->getPayment();
 
         // Init payments/details request
         $result = [];
 
-        if ($paymentData = $payment->getAdditionalInformation("threeDS2PaymentData")) {
+        if ($paymentData = $payment->getAdditionalInformation("adyenPaymentData")) {
             // Add payment data into the request object
             $request = [
-                "paymentData" => $payment->getAdditionalInformation("threeDS2PaymentData")
+                "paymentData" => $payment->getAdditionalInformation("adyenPaymentData")
```

Comment:
The data is already retrieved into the $paymentData var
```suggestion
                "paymentData" => $paymentData
```